athena-hbase/src/main/java/com/amazonaws/athena/connectors/hbase/HbaseEnvironmentProperties.java

```diff
@@ -0,0 +1,44 @@
+/*-
+ * #%L
+ * Amazon Athena Query Federation SDK
+ * %%
+ * Copyright (C) 2019 - 2024 Amazon Web Services
+ * %%
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ * #L%
+ */
+package com.amazonaws.athena.connectors.hbase;
+
+import com.amazonaws.athena.connector.lambda.connection.EnvironmentProperties;
+
+import java.util.HashMap;
+import java.util.Map;
+
+import static com.amazonaws.athena.connector.lambda.connection.EnvironmentConstants.DEFAULT_HBASE;
```

Comment:
I asked this before; just wondering now that I'm seeing this here if we have `default_XXX` for all connectors?

Reply:
no we do not. i try to re-use, but if connector has different default_connection_string name then I created another variable